Windows.Kits.cmake

```diff
@@ -100,8 +100,9 @@ set(CMAKE_RC_FLAGS_INIT "/nologo")
 set(MIDL_COMPILER "${WINDOWS_KITS_BIN_PATH}/${CMAKE_VS_PLATFORM_TOOLSET_HOST_ARCHITECTURE}/midl.exe")
 set(MDMERGE_TOOL "${WINDOWS_KITS_BIN_PATH}/${CMAKE_VS_PLATFORM_TOOLSET_HOST_ARCHITECTURE}/mdmerge.exe")
 
-# Add the 'Support' folder to the search path so that projects can opt-in to other functionality
-set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH};${CMAKE_CURRENT_LIST_DIR}/Support")
+# MSVC_TOOLCHAIN_DIR can be used to opt-in to other functionality by including other files in the toolchain directory.
+set(MSVC_TOOLCHAIN_DIR ${CMAKE_CURRENT_LIST_DIR} CACHE FILEPATH "MSVC toolchain directory")
```

Comment:
The `MSVC_` prefix gives me a little hesitation. For example, #18 calls out a possibility of toolchains for other compilers. I'm not sure that it's necessarily appropriate to grow the scope to that level yet, but the repo does contain MSVC and Clang toolchains. The repo does have a, er, 'branding problem' being called 'Toolchain', but just being `TOOLCHAIN_SUPPORT_DIR` would get my vote.

Reply:
I don't mind changing it, but I am afraid choosing `TOOLCHAIN_` makes people confused as it is such a general name. It would help if you chose a better name. For example, `WINDOWS_` seems like a good option.

Reply:
I was thinking that `TOOLCHAIN_` was sufficient since CMake only supports a single toolchain (doesn't it?) and consumers would/should use other discriminators (like, say, `CMAKE_CXX_COMPILER_ID`) to know that they're in a scope where they can use their toolchain support files. But `WINDOWS_TOOLCHAIN_` seems reasonable.

Aside; perhaps I should rename the GitHub repo to be "WindowsToolchain"? I don't know the implications of that. Perhaps making a new repo and migrating to that would be easier...?

Reply:
CMake allows multiple toolchains. For example, I can use the vcpkg toolchain with this toolchain.

Changing the repository name is safe. The URLs are automatically forwarded. Nothing breaks.